src/substrate/poller.ts

```diff
@@ -73,30 +73,37 @@ export class Poller extends IEventPoller<ApiPromise, Block> {
 
   /**
    * Connects to chain, fetches blocks specified in given range in provided batch size,
-   * prcoesses the blocks if a handler is provided else returns the blocks for 
-   * further processing
+   * prcoesses the blocks if a handler is provided 
    * @param range IDisconnectedRange having startBlock and optional endBlock
    * @param batchSize size of the batch in which blocks are to be fetched from chain
    * @param processBlockFn an optional function to process the blocks
    */
-  public async archive(range: IDisconnectedRange, batchSize: number = 500, processBlockFn: (block: Block) => any = null): Promise<Block[]> {
-    if(!range.endBlock){
+  public async archive(range: IDisconnectedRange, batchSize: number = 10, processBlockFn: (block: Block) => any = null) {
```

Comment:
Any reason why we're using a default batch of 10 now?

Reply:
reverted back to 500. While testing reduced the batch size to 10.